DirectProgramming/DPC++FPGA/Tutorials/Features/mem_channels/README.md

```diff
@@ -0,0 +1,310 @@
+| What you will learn               | How and when to use the `mem_channel` buffer property and the `-Xsno-interleaving` flag
+| Time to complete                  | 30 minutes
+
+
+
+## Purpose
+
+This FPGA tutorial demonstrates an example of using the `mem_channel` buffer
+property in conjuction with the `-Xsno-interleaving` flag to reduce the amount
+of resources required to implement a DPC++ FPGA design.
+
+By default, the Intel® oneAPI DPC++ compiler configures each global memory type
+in a burst-interleaved manner where memory words are interleaved across the
+available memory channels. This usually leads to better throughput because it
+prevents load imbalance by ensuring that memory accesses do not favor one
+external memory channel over another. Moreover, burst-interleaving enables you
+to view a multi-channel memory as one large channel so you can write your code
+without worrying about where each buffer should be allocated. However, this
+configuration can be expensive in terms of FPGA resources because the global
+memory interconnect required to orchestrate the memory accesses across all the
+channels is complex. For more information about burst-interleaving, please
+refer to the [oneAPI DPC++ FPGA Optimization
+Guide](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide).
+
+The Intel® oneAPI DPC++ compiler allows you to avoid this area overhead by
+disabling burst-interleaving and assigning buffers to individual channels. There
+are two advantages to such configuration:
+1. A simpler global memory interconnect is built, which requires a smaller
+   amount of FPGA resources than the interconnect needed for the
+   burst-interleaving configuration.
+2. Potential improvements to the global memory bandwidth utilization due to
+   less contention at each memory channel.
+
+Burst-interleaving should only be disabled in situations where satisfactory
+load balancing can be achived by assigning buffers to individual channels.
+Otherwise, the global memory bandwidth utilization may be reduced, which will
+negatively impact the throughput of your design. 
+
+To disable burst-interleaving, you need to pass the `-Xsno-interleaving` flag
+to your `dpcpp` command. With interleaving disabled, you can now specify, using
+the `mem_channel` property, in which memory channel each buffer should be
+allocated:
+```c++
+buffer a_buf(a_vec, {property::buffer::mem_channel{1}});
+buffer b_buf(b_vec, {property::buffer::mem_channel{2}});
+```
+The channels are assumed to have IDs `1,2,...,N` where `N` is the number of
+available channels.
+
+For FPGA boards that have multiple memory types, it is possible to select which
+memory you want to disable burst-interleaving for by passing the memory type to
+the `-Xsno-interleaving` flag: `-Xsno-interleaving=<global_memory_type>`. The
+memory type is usually indicated in the board specification XML file.
```

Comment:
'usually'?  If it's not in the XML file, then where is it?

Should we add a note about where to find the board specification XML file?  Something like 'The board specification XML file (boardspec.xml <or whatever the actual name is> can be found in the root directory of your BSP.  For example, for the PAC A10 board, you can find it . . blah blah"

Reply:
Addressed.